obs_img_utils/cli.py

```diff
@@ -177,17 +179,50 @@ def main(context):
     is_flag=True,
     help='Skip the image checksum validation.'
 )
+@click.option(
+    '--add-conditions-file',
+    type=click.STRING,
+    help='Specify conditions for the image from a file (witht the conditions '
```

Comment:
s/witht/with

Reply:
fixed! Installed a spell checker to detect typos before pushing the code to the repo... :sweat: